testsuite/forge-cli/src/main.rs

```diff
@@ -599,7 +599,8 @@ fn get_land_blocking_test(
 ) -> Option<ForgeConfig> {
     let test = match test_name {
         "land_blocking" => land_blocking_test_suite(duration), // TODO: remove land_blocking, superseded by below
-        "realistic_env_max_load" => realistic_env_max_load_test(duration, test_cmd, 7, 5),
+        // TODO: revert me before landing!!!
+        "realistic_env_max_load" => realistic_network_tuned_for_throughput_test(), // realistic_env_max_load_test(duration, test_cmd, 7, 5),
```

Comment:
Needs to be reverted

Reply:
Thanks boss! That's some MVP action 😄

Reply:
I accidentally took a glance at the test TPS and thought "that's too high" :P